app/mailers/verification_mailer.rb

```diff
@@ -1,10 +1,13 @@
 class VerificationMailer < ApplicationMailer
-  default from: -> { "#{ApplicationConfig['COMMUNITY_NAME']} Email Verification <#{SiteConfig.email_addresses[:default]}>" }
+  default from: lambda {
```

Comment:
Did Rails or Rubycop recommended this syntax switch, or did this just get changed unrelatedly? Asking since it seems like we have slightly more instances of the `->` lambda syntax throughout the code base as it stands currently

Reply:
I didn't even notice, Rubocop changed it :)